installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java

```diff
@@ -65,11 +64,12 @@ public void panelActivate() {
    private static long calculateAggregateSize(String sizes)
    {
       long totalSize = 0;
-      if (sizes != null && !sizes.isEmpty()) {
+
+      if (sizes != null && !sizes.trim().isEmpty() && !sizes.equals("null")) {
```

Comment:
This feels wrong, did we miss null check somewhere while reading from json file?